compiler/src/iree/compiler/Codegen/Common/CombineLayoutTransformation.cpp

```diff
@@ -278,6 +442,58 @@ insertIdentityMapScatter(RewriterBase &rewriter,
   return mapScatterOp;
 }
 
+static SmallVector<DistributionConfig>
+defaultWorkgroupDistributionConfigFn(ArrayRef<int64_t> sizes,
+                                     MLIRContext *ctx) {
+  DistributionConfig workgroupDistributionConfig;
+  workgroupDistributionConfig.tileSizes = SmallVector<int64_t>(sizes.size(), 1);
+  workgroupDistributionConfig.tileSizes.back() = 64;
+  workgroupDistributionConfig.mapping = llvm::map_to_vector(
+      llvm::seq<int64_t>(sizes.size()), [&](int64_t dim) -> Attribute {
+        switch (dim) {
+        case 0:
+        case 1:
+        case 2:
+          return IREE::Codegen::WorkgroupMappingAttr::get(
+              ctx, IREE::Codegen::symbolizeWorkgroupId(dim).value());
+        default:
+          return IREE::Codegen::WorkgroupMappingAttr::get(
+              ctx, IREE::Codegen::WorkgroupId::IdZ, dim - 2);
+        }
+      });
+  std::reverse(workgroupDistributionConfig.mapping.begin(),
+               workgroupDistributionConfig.mapping.end());
+  return {workgroupDistributionConfig};
+}
```

Comment:
If it becomes a testing pass, maybe move it to the main function (i.e., `runOnOperation`) and make it a lambda.

You also need to document in Passes.td that the distribution is using WorkgroupMappingAttr.